### PR TITLE
RSH1b5 New push.admin.device_registrations.removeWhere

### PR DESCRIPTION
**Pull request by jdavid**

### CODE REVIEW

**Comment by jdavid** [comment]
The push tests fail with the `dev` and `idempotent-dev` environments, so I've changed to the `sandbox` environment.

Then the idempotent tests fail since idempotency is not supported in `sandbox`, so I skip those tests if in the `sandbox` environment.

**Comment by jdavid** [comment]
ping @paddybyers

File: tests/ChannelIdempotentTest.php [hunk 1]
```diff
@@ -139,6 +142,9 @@ public function testIdempotentMixedIds() {
      * RSL1k4
      */
     public function testIdempotentLibraryGeneratedPublish() {
+        if (self::$testApp->getOptions()['environment'] == 'sandbox') {
+            $this->markTestSkipped('idempotent not available in sandbox environment');
```
Comment (mattheworiordan) [inline tests/ChannelIdempotentTest.php]: Idempotent is available in sandbox though. @paddybyers this is correct right?

Comment (jdavid) [reply tests/ChannelIdempotentTest.php]: yes, I've reverted that change and idempotent is working in the sandbox

**Comment by jdavid** [comment]
Not skipping idempotent tests in sandbox now.

Can I merge this? @mattheworiordan @paddybyers

**Comment by jdavid** [comment]
Don't know. This same test was failing around December if I remember correctly, @paddybyers looked at it as well but didn't find why. Then the failure disappeared, and now it's back.

**Pull request merged**